src/components/AddUsersModal/index.jsx

```diff
@@ -0,0 +1,316 @@
+              normalize={this.normalizeFileUpload}
+            />
+            <H3>Email Template</H3>
+            <Field
+              id="email-template-greeting"
+              name="email-template-greeting"
+              component={TextAreaAutoSize}
+              label="Customize Greeting"
+              onChange={this.handleFieldOnChange}
+            />
+            <Field
+              id="email-template-body"
+              name="email-template-body"
+              component={TextAreaAutoSize}
+              label="Body"
+              disabled
+            />
+            <Field
+              id="email-template-closing"
+              name="email-template-closing"
+              component={TextAreaAutoSize}
+              label="Customize Closing"
+              onChange={this.handleFieldOnChange}
+            />
+          </div>
+        </form>
+      </React.Fragment>
+    );
+  }
+
+  renderErrorMessage() {
+    const { error } = this.props;
+
+    return (
+      <div
+        ref={this.errorMessageRef}
+        tabIndex="-1"
+      >
+        <StatusAlert
+          alertType="danger"
+          iconClassName="fa fa-times-circle"
+          title="Unable to subscribe users"
+          message={error.length > 1 ? (
+            <ul className="m-0 pl-4">
+              {error.map(message => <li key={message}>{message}</li>)}
+            </ul>
+          ) : (
+              error[0]
+            )}
+        />
+      </div>
+    );
+  }
+
+  renderTitle() {
+    const { title } = this.props;
+
+    return (
+      <React.Fragment>
+        <span className="d-block">{title}</span>
+        <small>Add Subscriptions</small>
+      </React.Fragment>
+    );
+  }
+
+  render() {
+    const {
+      onClose,
+      submitting,
+      handleSubmit,
+    } = this.props;
+
+    return (
+      <React.Fragment>
+        <Modal
+          ref={this.modalRef}
+          title={this.renderTitle()}
+          body={this.renderBody()}
+          buttons={[
+            <Button
+              key="subscribe-users-submit-btn"
+              disabled={submitting}
+              className="subscribe-users-save-btn btn-primary"
+              onClick={handleSubmit(this.handleModalSubmit)}
+            >
+              <React.Fragment>
+                {submitting && <Icon className="fa fa-spinner fa-spin mr-2" />}
+                {'Assign License(s)'}
+              </React.Fragment>
+            </Button>,
+          ]}
+          closeText="Cancel"
+          onClose={onClose}
+          open
+        />
+      </React.Fragment>
+    );
+  }
+}
+
+AddUsersModal.defaultProps = {
+  error: null,
+  enterpriseId: null,
+};
+
+AddUsersModal.propTypes = {
+  // props From redux-form
+  handleSubmit: PropTypes.func.isRequired,
+  submitting: PropTypes.bool.isRequired,
+  submitSucceeded: PropTypes.bool.isRequired,
+  submitFailed: PropTypes.bool.isRequired,
+  error: PropTypes.arrayOf(PropTypes.string),
+
+  // custom props
```

Comment:
How is a license identified ... any licenseId?

Reply:
I think subscription license is per user base so it can be identified via the user's email/username. But I think @brittneyexline  or @adamstankiewicz would have a better idea.

Reply:
Here's the link to the License model in license-manager: https://github.com/edx/license-manager/blob/master/license_manager/apps/subscriptions/models.py#L110 
There is a UUID field